src/components/RecordTypesSelect/components/CompareRecordSelect.js

```diff
@@ -59,7 +59,7 @@ export const CompareRecordSelect = memo(({
   }, []);
 
   useEffect(() => {
-    const selectedExistingRecordElem = getElement(`#${existingRecord.type}`, existingRecordsElemRef.current);
+    const selectedExistingRecordElem = getElement(`[data-id=${existingRecord.type}]`, existingRecordsElemRef.current);
```

Comment:
Let's rename it to `data-testid` to be able to test it with `RTL` (see doc - https://testing-library.com/docs/dom-testing-library/api-queries#bytestid)

Reply:
This data-id attribute uses for finding DOM element, and not for testing component. So, I suggest not to change this one to `data-testid` to avoid misunderstanding.